shortfin/python/shortfin_apps/llm/components/kvcache/trie_attention_cache.py

```diff
@@ -90,6 +90,10 @@ def __eq__(self, other: object) -> bool:
         """Nodes are equal only if they are the same object."""
         return self is other
 
+    def __lt__(self, other):
+        """Sort nodes by their memory address."""
+        return id(self) < id(other)
```

Comment:
If the tests passed on Linux without this and they still fail on Windows with it... maybe this isn't the right implementation 🤔 

Could leave this off and just keep the test skipped.